pkgs/watcher/lib/src/directory_watcher/windows.dart

```diff
@@ -198,7 +198,11 @@ class _WindowsDirectoryWatcher
           });
           subscription.onError((Object e, StackTrace stackTrace) {
             _listSubscriptions.remove(subscription);
-            _emitError(e, stackTrace);
+            // "Path not found" can be caused by creating then quickly removing
+            // a directory: continue without reporting an error.
+            if (e is! PathNotFoundException) {
```

Comment:
I have no idea about this code, but can this not hide "real" path not found exceptions? Or is that impossible?

Reply:
Thanks--done.